### DISCUSSION TITLE
Plot times for 7713 Epyc

### DISCUSSION THREAD

**Discussion by grobalt** [madMAx43v3r/chia-plotter#245] [Show and tell]
Single socket, octa-channel memory 128GB.
1 plot , no parallel plotting
P2 sucks in scaling due to bad nvme (adata sx8200 512GB, always at 100% load)

![image](https://user-images.githubusercontent.com/79084968/121684620-36affe80-cabf-11eb-8ec4-83c3feaa1a73.png)

**Comment by madMAx43v3r** [comment]
yes P2 is heavy on `tmpdir` ...

**Reply by grobalt** [reply]
waiting for my ordered 1735 samsung nvme ... should improve a bit ;)

**Reply by polkadotbabe** [reply]
WTF? (Pardon my language guys!)
See: https://github.com/madMAx43v3r/chia-plotter/discussions/254

Dell R610, dual X5650, one hour and 45 seconds to make a plot, no ramdisk.

I've been seeing drastically faster-than-expected times out of Intel CPUs vis-a-vis AMDs with the chia plotter, seems even more so with @madMAx43v3r's plotter?

The speed diff is just crazy:
https://www.cpubenchmark.net/compare/AMD-EPYC-7713-vs-Intel-Xeon-X5650/4206vs1304
